run-qa-checks

```diff
@@ -7,3 +7,14 @@ openwisp-qa-check --skip-checkmigrations
 # test sphinx docs
 mkdir -p docs/source/_static
 make -C docs html
+
+PYTHON_VERSION=$(python -c 'import sys; print(".".join(map(str, sys.version_info[:2])))')
+if [[ $PYTHON_VERSION != 3.6 ]]; then
```

Comment:
let's try to run it without this if and see if it works

Reply:
Doesn't work for python3.6. We should keep this.

Reply:
bummer :)